vhdbuilder/packer/vhd-image-builder-mariner.json

```diff
@@ -483,6 +483,31 @@
       "source": "parts/linux/cloud-init/artifacts/containerd.service",
       "destination": "/home/packer/containerd.service"
     },
+    {
```

Comment:
should these changes also be added to the file `AgentBaker/vhdbuilder/packer/vhd-image-builder-mariner-arm64.json`

Reply:
No. We don't plan to support aarch64 for kata-cc at this moment.